1-mandatory/1-writers.js

```diff
@@ -62,7 +66,14 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  let deadWritersIn40s = writers.filter((writer) => {
+    return writer.age > 39 && writer.age < 50 && !writer.alive;
+  });
+  deadWritersIn40s.map((writer) => {
```

Comment:
Issue: Same as above `forEach` is more suitable than `map` here.

Reply:
changed